File: Lista de Exercícios (C)/C/Lista 3 - Sala/4.c
#include <stdio.h>

int main()
{
    float N;
    printf("Número menor ou igual a 50 e seu valor por uma multiplicação sucessiva por 3, antes que chegue a ser 250\n\n");

    printf("--Valores decimais devem ser digitados com (.)!--\n");
    printf("Número: "); scanf("%f", &N);
    printf("\n");
    
    if (N<=50){
        do{
            N=N*3;
        }while (N<250);
    }N=N/3;
    printf("%.2f\n", N);
}

File: Listas de Exercícios/Python/Lista 3 - Sala/4.py
print("Número menor ou igual a 50 e seu valor por uma multiplicação sucessiva por 3, antes que chegue a ser 250\n")

print("--Valores decimais devem ser digitados com (.)!--")
N=float(input("Número: "))

if (N<=50):
    while True:
        N=N*3
        if (N>=250):
            N=N/3
            break

print(f"\n{N}\n")